src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -204,3 +204,437 @@
+"""
+
+helps['cosmosdb sql container list'] = """
+type: command
+short-summary: List the SQL containers under an Azure Cosmos DB SQL database.
+"""
+
+helps['cosmosdb sql container show'] = """
+type: command
+short-summary: Show the details/properties of a SQL container under an Azure Cosmos DB SQL database.
+"""
+
+helps['cosmosdb sql container delete'] = """
+type: command
+short-summary: Delete the SQL container under an Azure Cosmos DB SQL database.
+"""
+
+# SQL database throughput
+helps['cosmosdb sql database throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL database throughput.
+"""
+
+helps['cosmosdb sql database throughput show'] = """
+type: command
+short-summary: Get the throughput of the SQL database.
+"""
+
+helps['cosmosdb sql database throughput update'] = """
+type: command
+short-summary: Update the throughput of the SQL database.
+"""
+
+# SQL container throughput
+helps['cosmosdb sql container throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL container throughput.
+"""
+
+helps['cosmosdb sql container throughput show'] = """
+type: command
+short-summary: Get the throughput of the SQL container.
+"""
+
+helps['cosmosdb sql container throughput update'] = """
+type: command
+short-summary: Update the throughput of the SQL container.
+"""
+
+helps['cosmosdb mongodb'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB resources.
+"""
+
+# MongoDB database
+helps['cosmosdb mongodb database'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB databases.
+"""
+
+helps['cosmosdb mongodb database create'] = """
+type: command
+short-summary: Create an MongoDB database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database list'] = """
+type: command
+short-summary: List the MongoDB databases under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database show'] = """
+type: command
+short-summary: Show the details/properties of a MongoDB database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database delete'] = """
+type: command
+short-summary: Delete the MongoDB database under an Azure Cosmos DB database account.
+"""
+
+# MongoDB collection
+helps['cosmosdb mongodb collection'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB collections.
+"""
+
+helps['cosmosdb mongodb collection create'] = """
+type: command
+short-summary: Create an MongoDB collection under an Azure Cosmos DB MongoDB database.
+examples:
+  - name: Create an Azure Cosmos DB MongoDB collection.
+    text: az cosmosdb mongodb collection create -g MyResourceGroup -a MyAccount -d MyDatabase -n MyCollection --shard "ShardingKey" --idx @indexes-file.json --throughput "500"
+    crafted: true
+"""
+
+helps['cosmosdb mongodb collection update'] = """
+type: command
+short-summary: Update an MongoDB collection under an Azure Cosmos DB MongoDB database.
```

Comment:
It should be under an Azure Cosmos DB database account or Azure Cosmos DB MongoDB database? Please check your description.

Reply:
Current is correct